tests/cephadm/test_service_stop.py

```diff
@@ -0,0 +1,31 @@
+from cli.ceph.orch.orch import Orch
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verify service stop operation for Mon and Mgr service
+    """
+    base_cmd = "cephadm shell -- ceph"
+    service_name = ["mgr", "mon"]
+
+    # Stop the service
+    node = ceph_cluster.get_nodes(role="installer")[0]
+    orch = Orch(node, base_cmd)
+    for service in service_name:
+        log.info(f"Stopping {service} service")
+        try:
+            out = orch.stop(service)
+            log.info(f"Stopping {service} service operation executed")
+            expected_message = f"Stopping entire {service} service is prohibited."
+            if expected_message in out:
```

Comment:
In case of command failure, `cephadm.orch.stop` function wont return any output. It will raise an `ceph.CommandFailed`  which needs to be catched and error need to be checked.

Reply:
Added the exception to catch command failure